cardano-api/src/Cardano/Api/ProtocolParameters.hs

```diff
@@ -475,13 +475,11 @@ instance HasTypeProxy UpdateProposal where
 instance HasTextEnvelope UpdateProposal where
     textEnvelopeType _ = "UpdateProposalShelley"
 
-instance ToCBOR UpdateProposal where
-    toCBOR = toCBOR . toShelleyUpdate @StandardShelley
```

Comment:
I wish I had noticed that this type was also changed from have a `FromCBOR` to an `Annotator FromCBOR` while I was working on https://github.com/input-output-hk/cardano-ledger-specs/pull/2180, but I did not notice it since consensus does not use this type. But I do not think it is worth updating cardano-ledger-specs and ourorboros-network again before merging this PR. Especially since we do not have a concrete plan for the what to do with the decoders for protocol parameters in alonzo yet.